app/assets/javascripts/listings/templates/listing/_panel-apply.html.slim

```diff
@@ -91,7 +91,9 @@
 
   .content-group.bg-snow
     p.c-charcoal.t-small
-      | Do not apply online and also send in a paper application. All applications containing any person who appears on more than one application will be removed from the lottery.
+      span ng-if="listing.Accepting_Online_Applications && !listing.Lottery_Results"
```

Comment:
@callaghanc ah, gotcha, makes sense that you copied the logic from there. Then the extra check is perhaps redundant in both of those cases, but I'll take a look at how the `_panel-apply` is getting loaded in the ng-include and make sure my understanding above is correct.